R/package.R

```diff
@@ -32,6 +32,9 @@ as_pkgdown <- function(pkg = ".", override = list()) {
   )
   meta <- modify_list(template_config, meta)
 
+  # BS version can also be set by template package itself
+  bs_version <- get_bootstrap_version(list(meta = meta))
```

Comment:
Given that this function is the only place that `get_bootstrap_version()` is used, I think it might be slightly clearer to do something like:

```R
# Update bs_version if set in template
bs_version <- bs_version %||% template_config$bootstrap 
```